Tools/Scripts/compare-results

```diff
@@ -349,6 +369,30 @@ def JetStream2Results(payload):
 
     return results
 
+def JetStream3Results(payload):
+    assert detectJetStream3(payload)
+
+    js = payload["JetStream3.0"]
+
+    iterations = 0
+    if "gaussian-blur" in js["tests"]:
```

Comment:
Why do we not just always do the else?

Reply:
That's updated in this patch https://github.com/WebKit/WebKit/pull/12534. When we run `run-benchmark` script with option `--subtests` the output json object doesn't have `gaussian-blur` entry is missing under `tests` entry.